### PR TITLE
Prevent UI mods from being able to make network changes

### PR DESCRIPTION
**Pull request by Garanas**

### CODE REVIEW

File: lua/ui/game/gamemain.lua [hunk 1]
```diff
@@ -182,16 +182,6 @@ function CreateUI(isReplay)
...
-    -- tweak networking parameters
-    ConExecute('net_MinResendDelay 100')        -- standard value of 100ms
-    ConExecute('net_MaxResendDelay 1000')       -- standard value of 1000ms
-
-    ConExecute('net_MaxSendRate 2048')          -- standard value of 1024 bytes
-    ConExecute('net_MaxBacklog 2048')           -- standard value of 1024 bytes
-
-    ConExecute('net_SendDelay 5')               -- standard value of 25ms
-    ConExecute('net_AckDelay 5')                -- standard value of 25ms
-
```
Comment (BlackYps) [inline lua/ui/game/gamemain.lua:194]: Are the network changes intentionally removed completely?

**Comment by 4z0t** [comment]
Isn't console bypasses this?

**Comment by Hdt80bro** [comment]
`ConExecuteSave` needs to be patched too then

**Comment by Garanas** [comment]
> `ConExecuteSave` needs to be patched too then

I'll take that into account

**Comment by Garanas** [comment]
> Isn't console bypasses this?

Perhaps, but that is limited to your game instance

**Pull request merged**